x/mint/types/types.proto

```diff
@@ -48,6 +57,4 @@ message Params {
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
     (gogoproto.nullable)   = false
   ];
-  // expected blocks per year
```

Comment:
We can't have another field eventually added here with the same field number. So we must either:

1. Comment it out
2. Leave it

Any suggestions @aaronc ?

Reply:
Maybe setting the deprecated annotation from protocbuf?

Reply:
You can just make it a protobuf `reserved` field with a comment.